docs/pages/configuration.rst

```diff
@@ -7,36 +7,17 @@ Configuration
    protocol:
      login:
        path: /login
-       allowed_parameters:
-         - service
-         - renew
-         - gateway
      serviceValidate:
        path: /p3/serviceValidate
-       allowed_parameters:
-         - service
-         - ticket
-         - pgtUrl
-         - renew
-         - format
        default_parameters:
          pgtUrl: https://my-app/casProxyCallback
      logout:
        path: /logout
-       allowed_parameters:
-         - service
        default_parameters:
          service: https://my-app/homepage
```

Comment:
Actually those names are directly taken from the [CAS protocol](https://apereo.github.io/cas/6.1.x/installation/Logout-Single-Signout.html): 

```
Per the [CAS Protocol](https://apereo.github.io/cas/6.1.x/protocol/CAS-Protocol.html), the /logout endpoint is responsible for destroying the current SSO session. Upon logout, it may also be desirable to redirect back to a service. This is controlled via specifying the redirect link via the service parameter. The specified service must be registered in the service registry of CAS and enabled and CAS must be allowed to follow service redirects.
```

I do agree that those names might be confusing, but adding new names would add another layer on top of what's already existing, which honestly, I prefer to avoid.